fmcib/utils/idc_helper.py

```
@@ -190,53 +190,90 @@ def download_RADIO(path, samples=None):
+        logger.error(f"Failed to read mask {mask_path}: {e}")
+        return None
 
-    # Get centroid from label shape filter
+    # Extract centroid from the mask
     label_shape_filter = sitk.LabelShapeStatisticsImageFilter()
     label_shape_filter.Execute(mask)
 
+    # Some masks label is 1, others are 255; try 255 first, else 1
     try:
         centroid = label_shape_filter.GetCentroid(255)
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)